### ISSUE TITLE
Disable accessibility features for SeparatorLineCell

### ISSUE THREAD

**Issue by oliverkrakora** [allaboutapps/DataSource#30]
[SeparatorLineCell](https://github.com/allaboutapps/DataSource/blob/master/DataSource/SeparatedSection/SeparatorLineCell.swift) should be marked as not accessible so it is ignored by VoiceOver like a normal system separator.
